### ISSUE TITLE
Thumbnail images are missing alt text attribute

### ISSUE THREAD

**Issue by srappel** [UWM-Libraries/GeoDiscovery#317]
Thumbnail images rendered by the Sidecar Images gem do not include alt text.

Alt text should likely be empty as the thumbnails are decorative and should be skipped by screen readers.

<img width="493" height="502" alt="Image" src="https://github.com/user-attachments/assets/a80662cb-12a9-4f98-bb65-14636dbd7a2c" />
